### PR TITLE
Feature/pdct 1110 only show entities belonging to user orgcorpus

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

Only show families that belong to the same org as user unless superuser.

[PDCT-1110](https://linear.app/climate-policy-radar/issue/PDCT-1110/only-show-families-belonging-to-user-orgcorpus)

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/repository/family.py [hunk 1]
```diff
@@ -116,14 +116,22 @@ def _update_intention(
     return update_title, update_basics, update_metadata, update_collections
 
 
-def all(db: Session) -> list[FamilyReadDTO]:
+def all(db: Session, org_id: int, is_superuser: bool) -> list[FamilyReadDTO]:
```
Comment (diversemix) [inline app/repository/family.py:119]: Just a thought ... I'm wondering if we could pass a `UserContext` object around at some point with these (and other) attribute in?

**Review by diversemix — APPROVED** [review]
Great stuff :fireworks:

File: app/repository/family.py (hunk 1 above)
Comment (katybaulch) [reply app/repository/family.py:119]: Yeah I like that idea, a lot less janky!

Comment (katybaulch) [reply app/repository/family.py:119]: https://linear.app/climate-policy-radar/issue/PDCT-1113/pass-usercontext-object-around-instead-of-user-properties-individually

**Pull request merged**